css-view-transitions-1/Overview.bs

```diff
@@ -1656,6 +1660,20 @@ urlPrefix: https://html.spec.whatwg.org/multipage/rendering.html; type: dfn;
 
 			1. Otherwise:
 
+				1. If any of the following is true:
+
+					- Any [=flat tree=] ancestor of |capturedElement|'s [=new element=] [=skips its contents=].
```

Comment:
@vmpstr, can you review this? It's consistent with the resolution [here](https://github.com/w3c/csswg-drafts/issues/8060#issuecomment-1411189366): "Element with c-v: auto continues to stay relevant to the user if it has descendants which participate in a transition."

Since this element is participating in a transitions we'll ensure the element, and all its ancestors, stay relevant to the user. I'm not sure about cases like c-v: hidden. We're saying that if the developer toggles their CSS to do this, we'll abort the transition. It seems reasonable but we haven't explicitly resolved on this.

I was trying to think whether this lines up with the resolution [here](https://github.com/w3c/csswg-drafts/issues/7882#issuecomment-1377660423): "The element must have an associated box". But no, hidden elements do generate boxes. That case is covered by the next line.

Reply:
The text here sounds good to me. The `content-visibility: hidden` value which will skip its contents regardless of relevance to the user should be generally treated the same as `display: none`.

You're right that the box generation is not sufficient here, so having an explicit point about skipping contents is definitely the right thing to say.

Reply:
Ok cool. For my point above about getting an explicit resolution on this, #7882 was meant to be a generic resolution that anything we check during capture old/new state must not change during the animation. If it does then we can abort. And we check for both the things in capture old/new state:

```
			1. If any [=flat tree=] ancestor of this |element| [=skips its contents=], then [=continue=].
			1. If |transitionName| is ''view-transition-name/none'',
				or |element| is [=element-not-rendered|not rendered=],
				then [=continue=].
```

So LGTM for me too.